@xen-orchestra/lite/src/locales/fr.json

```diff
@@ -25,6 +25,7 @@
   "back-pool-dashboard": "Revenez au tableau de bord de votre pool",
   "backup": "Sauvegarde",
   "cancel": "Annuler",
+  "cannot-login-from-slave": "Vous ne pouvez pas vous connecter à partir d'un hôte slave. {masterLink}",
```

Comment:
@olivierlambert is `slave/master` need to be translated? Should we use `primary/secondary` instead?

Reply:
As discussed with Olivier, in FR, use: `Primaire/secondaire`

Reply:
Only in french? We keep master/slave naming in english?

Reply:
For the moment, yes.